core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.jelly

```diff
@@ -0,0 +1,24 @@
+<!--
+The MIT License
+Copyright (c) 2025, Stefan Spieker
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <st:redirect url="../manage/cloud/" />
```

Comment:
> I also used a curl command `curl -vL` to verify that it is working:

Due to `..` use here instead of `rootURL` as before (and the insanity coming from #6126), the result is a little awkward with `http://localhost:8090/manage/configureClouds` which was the most likely URL.

<img width="1032" height="113" alt="Screenshot 2025-08-17 at 22 29 20" src="https://github.com/user-attachments/assets/8a609bf5-49d1-414b-b1cf-27e33e49fcaa" />